etc/config/circt.amazon.properties

```diff
@@ -7,8 +7,9 @@ supportsExecute=false
 supportsAsmDocs=false
 
 group.circtopt.compilers=circtopt-trunk
-group.circtopt.isSemVer=false
+group.circtopt.isSemVer=true
 group.circtopt.groupName=CIRCT opt
+group.circtopt.baseName=CIRCT opt
 
 compiler.circtopt-trunk.exe=/opt/compiler-explorer/circt-trunk/bin/circt-opt
-compiler.circtopt-trunk.name=CIRCT opt (trunk)
+compiler.circtopt-trunk.semver=(trunk)
```

Comment:
Extra change: Made this a semver compiler because the group in defaults was defining a baseName

Reply:
Sounds good